assets/javascripts/tests.js

```diff
@@ -191,16 +191,17 @@ function changeJobPrio(jobId, delta, linkElement) {
   }
 
   var newPrio = currentPrio + delta;
-  $.ajax({
-    url: urlWithBase('/api/v1/jobs/' + jobId + '/prio?prio=' + newPrio),
-    method: 'POST',
-    success: function (result) {
-      prioValueElement.text(newPrio);
-    },
-    error: function (xhr, ajaxOptions, thrownError) {
-      addFlash('danger', 'Unable to set the priority value of job ' + jobId + '.');
-    }
-  });
+
+  fetchWithCSRF(urlWithBase('/api/v1/jobs/' + jobId + '/prio?prio=' + newPrio), {method: 'POST'})
```

Comment:
If you like you can also change the code like this (but of course this is actually independent and could be done as a separate improvement):
```suggestion
  fetchWithCSRF(urlWithBase(`/api/v1/jobs/${jobId}/prio?prio=${newPrio}`), {method: 'POST'})
```